### PR TITLE
Multichannel config

### PR DESCRIPTION
**Pull request by mboussarov**
Explain what this PR does.

### CODE REVIEW

File: numalogic/transforms/_stateless.py [hunk 1]
```diff
@@ -156,3 +156,45 @@ def transform(self, X: npt.NDArray[float], **__) -> npt.NDArray[float]:
 
     def inverse_transform(self, X: npt.NDArray[float]) -> npt.NDArray[float]:
         return X.reshape(-1, self.n_features)
+
+
+class FlattenVectorWithPadding(StatelessTransformer):
+    """A stateless transformer that flattens some of the columns and does padding on the rest.
```
Comment (s0nicboOm) [inline numalogic/transforms/_stateless.py:162]: Can we have __slots__ defined here. We also might have to move 182 and 183 inside transform.

**Review by s0nicboOm — APPROVED** [review]
minor comment otherwise LGTM!

File: numalogic/transforms/_stateless.py (hunk 1 above)
Comment (mboussarov) [reply numalogic/transforms/_stateless.py:162]: Added

File: numalogic/transforms/_stateless.py [hunk 2]
```diff
@@ -156,3 +156,54 @@ def transform(self, X: npt.NDArray[float], **__) -> npt.NDArray[float]:
...
+    @staticmethod
+    def _feature_indexes(features_all: list[str], features: list[str]) -> list[int]:
+        return [features_all.index(f) for f in features]
+
+    def __init__(self, features: list[str], flatten_features: list[str], padding_with: float = 0.0):
+        self.features = features
+        self.flatten_features = flatten_features
+        self.padding_with = padding_with
+
+        self.padding_features = list(set(features) - set(flatten_features))
```
Comment (ab93) [inline numalogic/transforms/_stateless.py:190]: Can we raise an error here if `self.padding_features` is empty?

Comment (mboussarov) [reply numalogic/transforms/_stateless.py:190]: Updated, please validate and resolve

**Review by ab93 — APPROVED** [review]

**Pull request merged**